config.json

```diff
@@ -1430,6 +1430,19 @@
         "strings"
       ],
       "deprecated": true
+    },
+    {
+      "slug": "yacht",
+      "uuid": "77d0ab59-3824-4a79-9db3-e0bb35c6fc63",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Please pick one of the core exercises <3

Reply:
Done :)